.github/workflows/fast-forward.yml

```diff
@@ -0,0 +1,20 @@
+name: Fast forward merge
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  fast-forward:
+    if: ${{ github.event.label.name == 'fast-forward' && github.base_ref != 'main' }}
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Fast forwarding
+        uses: sequoia-pgp/fast-forward@ea7628bedcb0b0b96e94383ada458d812fca4979
+        # See https://docs.github.com/en/graphql/reference/enums#mergestatestatus
+        if: ${{ github.event.pull_request.mergeable_state  == 'clean' }}
```

Comment:
Should we print the instruction what to do if the `mergeable_state` != `clean` instead of just skipping the step?

Reply:
done